src/common/translation/i18n/fi.json

```diff
@@ -148,6 +148,91 @@
     },
     "skipToContentLabel": "Siirry suoraan sisältöön"
   },
+  "profile": {
+    "common": {
+      "message": {
+        "serviceDescription": "Hienoa että olet mukana Kulttuurin kummilapset -toiminnassa! Täällä voit tarkastella tapahtumakutsuja ja ilmoittautua tapahtumiin. Huomioithan, että kummilapsi voi osallistua vuosittain kahteen tapahtumaan, ellei toisin ilmoiteta. Nähdään kulttuurin parissa!"
```

Comment:
Should this be "Hienoa että" -> "Hienoa, että"

Reply:
Hmn, maybe 🤔 Can't recall these rules in enough detail. This is how the copy is in the design.

Reply:
Perhaps "Hienoa, että", unravels into: "On hienoa, että" which would suggest that the comma should be there.